### PR TITLE
L brierton patch 1

### PR DESCRIPTION
**Pull request by L-brierton**
Performed some user testing and updated the readme with my findings. Corrected minor spelling mistake in direWolf JSON file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
...
+* Direwolf
+* Flying Sword
+* Giant Eagle
+* Goblin
+* Green Dragon
+* Skeleton 
+* Troll
+
+## Output
+The results of the program wil be outputted to the console. Debugging information will be displayed in red text. The winner of the brawl and how many of the winning team will be shown, then the turn by turn battle information.
```
Comment (ConnorClancy) [inline README.md]: Typo in "wil". Also the if you look at BrawlerOutputter class the output can be returned as either a JSON object with a specific structure in getBrawlOutput(), or as a String in getBrawlOutputAsString(). Can you update to reflect this pls?

File: dnd-brawler-1/sampleJsonCombatants/direWolf.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-	"name": "Dire Wold",
+	"name": "Dire Wolf",
```
Comment (ConnorClancy) [inline dnd-brawler-1/sampleJsonCombatants/direWolf.json:2]: Thanks for spotting this, Can you also fix the spelling error in "crocodile.json" while you're at it ;)

**Review by ConnorClancy — CHANGES_REQUESTED** [review]
Thanks for doing all this, readme is super explicit and well researched. Left a couple comments for minor changes

File: dnd-brawler-1/sampleJsonCombatants/direWolf.json (hunk 2 above)
Comment (L-brierton) [reply dnd-brawler-1/sampleJsonCombatants/direWolf.json:2]: ...fine. But I'm doing it because I want to do it :P

File: README.md (hunk 1 above)
Comment (L-brierton) [reply README.md]: Good spot, thanks for that. Additional output information has been included.

File: dnd-brawler-1/sampleJsonCombatants/direWolf.json (hunk 2 above)
Comment (ConnorClancy) [reply dnd-brawler-1/sampleJsonCombatants/direWolf.json:2]: \#manipulated

**Review by ConnorClancy — APPROVED** [review]

**Pull request merged**